tests/integration/test_workspace.py

```diff
@@ -21,6 +21,22 @@ def test_workspace_upload_download_notebooks(w, random):
     w.workspace.delete(notebook)
 
 
+def test_workspace_unzip_notebooks(w, random):
```

Comment:
Can we add a test that calls `read()` with a parameter to test reading `N` bytes?

Reply:
Added as Unit Test